hack/common.sh

```diff
@@ -485,29 +485,49 @@ KUBE_GIT_VERSION='${KUBE_GIT_VERSION-}'
 EOF
 }
 
+# golang 1.5 wants `-X key=val`, but golang 1.4- REQUIRES `-X key val`
+os::build::ldflag() {
```

Comment:
It would be better to have implemented these flags identical for OpenShift and Kube.  Having one way for OpenShift flags and one way for Kube flags is worse than a single if block in ::ldflags